workers/unity/Packages/io.improbable.gdk.testutils/MockWorld.cs

```diff
@@ -45,11 +44,38 @@ public static MockWorld Create(Options options)
 
             mockWorld.Linker = new EntityGameObjectLinker(mockWorld.Worker.World);
 
+            mockWorld.GetSystem<CommandSystem>().SetSender(mockWorld.CommandSender);
+
             PlayerLoopUtils.ResolveSystemGroups(mockWorld.Worker.World);
 
             return mockWorld;
         }
 
+        public void Setup<TRequest>(uint componentId)
+        {
+            CommandSender.Setup<TRequest>(componentId);
+        }
+
+        public IEnumerable<long> GetOutboundCommandRequestIds<TRequest>() where TRequest : ICommandRequest
+        {
+            return CommandSender.GetOutboundCommandRequestIds<TRequest>();
+        }
+
+        public void GenerateResponses<TRequest, TResponse>(Func<CommandRequestId, TRequest, TResponse> creator)
+            where TRequest : ICommandRequest
+            where TResponse : struct, IReceivedCommandResponse
+        {
+            CommandSender.GenerateResponses(creator);
+        }
+
+        public void GenerateResponse<TRequest, TResponse>(CommandRequestId id,
+            Func<CommandRequestId, TRequest, TResponse> creator)
+            where TRequest : ICommandRequest
+            where TResponse : struct, IReceivedCommandResponse
+        {
+            CommandSender.GenerateResponse(id, creator);
+        }
```

Comment:
I don't think these methods belong here. Similarly to the `MockConnectionHandler` where we don't add passthroughs here we should make the `MockCommandSender` public and allow users to access it as: `World.CommandSender.GenerateResponses(...);`.